soroban-test-wasms/src/lib.rs

```diff
@@ -42,25 +42,22 @@
 //!      documentation purpopses without having to worry about breaking tests in
 //!      the host here.
 
-pub const ADD_I32: &'static [u8] =
-    include_bytes!("../wasm-workspace/opt/example_add_i32.wasm").as_slice();
-pub const CREATE_CONTRACT: &'static [u8] =
+pub const ADD_I32: &[u8] = include_bytes!("../wasm-workspace/opt/example_add_i32.wasm").as_slice();
```

Comment:
Interesting, I don't quite understand why static is gone here

Reply:
It's implied since the const lives for the whole program.